Server/src/main/java/org/openas2/app/partner/AddPartnerCommand.java

```diff
@@ -17,57 +23,82 @@
  *
  * @author joseph mcverry
  */
+
 public class AddPartnerCommand extends AliasedPartnershipsCommand {
+
+    private Log logger = LogFactory.getLog(getClass().getName());
+
     public String getDefaultDescription() {
         return "Add a new partner to partnership store.";
     }
 
     public String getDefaultName() {
+
         return "add";
     }
 
     public String getDefaultUsage() {
+
         return "add name <attribute 1=value-1> <attribute 2=value-2> ... <attribute n=value-n>";
     }
 
     public CommandResult execute(PartnershipFactory partFx, Object[] params) throws OpenAS2Exception {
+
+        logger.info("partFx=" + partFx.toString());
```

Comment:
Adding this at INFO level logging will proliferate large log files. It should be at TRACE or DEBUG level.